matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/homeserver/HomeServerCapabilitiesDataSource.kt

```diff
@@ -35,4 +40,14 @@ internal class HomeServerCapabilitiesDataSource @Inject constructor(
             }
         }
     }
+
+    fun getHomeServerCapabilitiesLive(): LiveData<Optional<HomeServerCapabilities>> {
+        val liveData = monarchy.findAllMappedWithChanges(
```

Comment:
I agree we could do that since the `lastUpdatedTimestamp` is not even exposed in the SDK Api via `HomeServerCapabilities` model. But do you know how to write such Realm query? I cannot find a way to implement this.